OpenRA.Mods.Common/Activities/Sell.cs

```diff
@@ -37,9 +37,9 @@ public override bool Tick(Actor self)
 			var sellValue = self.GetSellValue();
 
 			// Cast to long to avoid overflow when multiplying by the health
-			var hp = health != null ? (long)health.HP : 1L;
```

Comment:
This cast was added in #14651 - are you sure it is not needed anymore? If yes, should it be changed in the other places too?

Reply:
Unless there is an implicit cast I'm missing these should be added back

Reply:
Isn't the `1L` going to be that implicit cast?

Reply:
Ah I see, because one branch is a long the entire thing gets interpreted as long I assume.

Reply:
Kept it.